api/users.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express");
+
+const userController = require("../controllers/users");
+
+const router = express.Router();
+
+router.post("/signup", userController.register);
+router.post("/login", userController.login);
+router.get("/logout", userController.logout);
+router.get("/current", userController.current);
```

Comment:
Te dwie ścieżki powinny być authorized. To znaczy wylogować może się użytkownik który istnieje albo jest zalogowany. Jeśli dodasz tutaj `auth` wówczas w tych kontrolerach nie będziesz musiała pobierać danych ponownie ponieważ one już zostaną pobrane przez middleware. 
Ma sens?